<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QWidget" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Simple Chat Server</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(12, 95, 58);
color: rgb(255, 255, 255);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="serverIcon">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../Pictures/database.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>298</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(12, 95, 58);
font: 25 italic 14pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>Online now</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPlainTextEdit" name="logEditor">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="usersList">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(12, 95, 58);
font: 25 italic 14pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>Server Logger</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="startStopButton">
     <property name="styleSheet">
      <string notr="true">     background-color: rgb(12, 95, 58);
     border-style: outset;
     border-width: 2px;
     border-radius: 5px;
     border-color: beige;
     font: bold 14px;
	color: rgb(255, 255, 255);
     min-width: 10em;
     padding: 6px;
</string>
     </property>
     <property name="text">
      <string>Start Server</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="deleteButton">
     <property name="styleSheet">
      <string notr="true">     background-color: rgb(12, 95, 58);
     border-style: outset;
     border-width: 2px;
     border-radius: 5px;
     border-color: beige;
     font: bold 14px;
	color: rgb(255, 255, 255);
     min-width: 5em;
     padding: 6px;</string>
     </property>
     <property name="text">
      <string>Delete user</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
